pkg/daemon/update.go

```diff
@@ -814,9 +814,7 @@ func reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (*machineConfigDif
 			return nil, errors.New("ignition Passwd Groups section contains changes")
 		}
 		if !reflect.DeepEqual(oldIgn.Passwd.Users, newIgn.Passwd.Users) {
-			// check if the prior config is empty and that this is the first time running.
-			// if so, the SSHKey from the cluster config and user "core" must be added to machine config.
-			if len(oldIgn.Passwd.Users) > 0 && len(newIgn.Passwd.Users) >= 1 {
+			if len(newIgn.Passwd.Users) >= 1 {
```

Comment:
I just realized that you could delete the entire "core" user and we'd silently be ok with it. Maybe for safety purposes its worth checking here? (Doesn't have to be part of this PR, just wanted to note)

Reply:
this pr is related to users doing that via deleting the 99-xxx-ssh

Reply:
Oh wait, this PR is saying that if you delete the entire `core` user, you should clear the sshkeys.
Do we support removing the whole user? Its not like we actually delete the user from the system, so if you had e.g. provided a password, its still accessible

Reply:
ok discussed with jerry, and have a new path forward so will repush with some updates!